### PR TITLE
(#55) Enabled docker + integration tests in Travis build

### PR DESCRIPTION
**Pull request by llorllale**
This PR:
* solves #55 
* Uses as reference the [Travis Docs](https://docs.travis-ci.com/user/docker/)
* enables docker + integration tests in the Travis build

Note: the coverage provided by the integration tests will be included in the coveralls report.

### CODE REVIEW

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this pull request is too small, just 5 lines changed (less than 10), there will be no formal code review; in the future, try to make sure your pull requests are not too small; @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please review this and merge or reject

**Comment by llorllale** [comment]
@amihaiemil 0crat is a funny dude

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
+sudo: required
```
Comment (amihaiemil) [inline .travis.yml]: @llorllale Is sudo really required?

Comment (llorllale) [reply .travis.yml]: @amihaiemil I'm not entirely sure (haven't tested), but the [Travis Docs](https://docs.travis-ci.com/user/docker/) seem to include it always

Comment (amihaiemil) [reply .travis.yml]: @llorllale I see, let's leave it like that then

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order

**Pull request merged**